rerun_py/rerun_sdk/rerun/error_utils.py

```diff
@@ -115,7 +115,7 @@ def _send_warning_or_raise(
         # TODO(jleibs): Context/stack should be its own component.
         context_descriptor = _build_warning_context_string(skip_first=depth_to_user_code + 1)
 
-        log("rerun", TextLog(text=f"{message}\n{context_descriptor}", level="WARN"), recording=recording)  # NOLINT
```

Comment:
In the odd chance that someone is relying on this, could you add a note in the migration guide?